SU-HKKU/cardinal_service_profile
# Added back to top link and animation

# READY FOR REVIEW

# Summary
- Added back to top link on the bottom right.

# Need Review By (Date)
- next week.

# Urgency
- low

# Steps to Test
1. checkout this branch
1. clear caches and browser caches
1. view a long page, like the opportunity list page
1. scroll down and validate the link appears and functions like expected.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
